shopping-cart/app/shoppingcart/common/ExtendedLineItemBean.java

```diff
@@ -0,0 +1,36 @@
+package shoppingcart.common;
+
+import common.contexts.UserContext;
+import common.controllers.ReverseRouter;
+import common.models.AttributeBean;
+import common.models.LineItemBean;
+import common.models.ProductDataConfig;
+import io.sphere.sdk.carts.LineItem;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public class ExtendedLineItemBean extends LineItemBean {
```

Comment:
I am still confused about the purpose and structure of LineItemBean and ExtendedLineItemBean.
Can you document them and/or give them better names?